lib/msal-angular/src/msal.service.ts

```diff
@@ -83,5 +84,7 @@ export class MsalService implements IMsalService {
         this.logger = logger.clone(name, version);
         this.instance.setLogger(logger);
     }
-
+    getConfiguration(): BrowserConfiguration {
```

Comment:
I'm okay with not including this, given it doesn't add or changing anything, and a component can invoke any of the functions on the instance directly if it wants.